modules/admin_manual/pages/installation/manual_installation.adoc

```diff
@@ -85,8 +85,8 @@ However, if you want or need to install it, then we suggest the following steps:
 
 ----
 wget http://download.opensuse.org/repositories/server:/php:/extensions/SLE_12_SP1/ server:php:extensions.repo -O /etc/zypp/repos.d/memcached.repo
-zypper refresh
-zypper install php5-APCu
+sudo zypper refresh
```

Comment:
IMHO this is the wrong approach, you should just mention to execute these commands as root. The current state is even really inconsistent because writing the repo file will fail.

Reply:
Sudo's a more secure and trackable approach, no?

Reply:
Why is it more secure? You are anyway executing the commands as root. But even if you keep sudo, `wget http://download.opensuse.org/repositories/server:/php:/extensions/SLE_12_SP1/ server:php:extensions.repo -O /etc/zypp/repos.d/memcached.repo` won't work as non-root user.